README.md

```diff
@@ -44,6 +44,7 @@ OpenSearch Continuous Integration is an open source CI system for OpenSearch and
 6. Log onto the AWS Console of the account, navigate to [cloud watch](https://console.aws.amazon.com/cloudwatch/home), open log groups, looking for `JenkinsMainNode/var/log/jenkins/jenkins.log`
 7. Search the logs for `Jenkins initial setup is required. An admin user has been created and a password generated.` After that entry the password for the jenkins instance will be in the cloudwatch logs.
 8. Go to the `CI-Dev.JenkinsExternalLoadBalancerDns` url returned by CDK output to access the jenkins host.
+9. While destroying the stack make sure you delete the agent nodes manually (via jenkins UI or AWS console) so that shared resources (like vpc, security groups, etc) can be deleted.
```

Comment:
Switch to -> `If you destroy the stack,` I never destroy my stacks, and I wouldn't recommend it to most developers unless you are doing specific setup where you need a fresh environment.